tools/src/format.rs

```diff
@@ -129,13 +128,16 @@ pub fn format_code(jumps: &HashMap<String, Jump>, format: FormatAs,
                     let syms = item.sym.split(',');
                     for sym in syms {
                         match jumps.get(sym) {
-                            Some(jump) => {
-                                if !(&jump.path == path && jump.lineno == cur_line) {
-                                    let key = format!("{}:{}", jump.path, jump.lineno);
-                                    let mut obj = json::Object::new();
-                                    obj.insert("sym".to_string(), Json::String(sym.to_string()));
-                                    obj.insert("pretty".to_string(), Json::String(jump.pretty.clone()));
-                                    menu_jumps.insert(key, Json::Object(obj));
+                            Some(jump_list) => {
```

Comment:
pre-existing, but instead of using a match on jumps.get(sym) you can use the `if let` construct to reduce one level of nesting:

`if let Some(jump_list) = jumps.get(sym) {`